widget/playground/src/containers/StyleLayout/StyleLayout.CustomColors.tsx

```diff
@@ -175,14 +218,12 @@ export function CustomColorsSection(props: CustomColorsTypes) {
                 placeholder={widgetColor.label}
                 color={getMainColor(widgetColor.key, tab, theme) || ''}
                 onChangeColor={(color) =>
-                  onChangeColors(widgetColor.key, tab, color)
+                  onChangeColor(widgetColor.key, tab, color)
                 }
                 onReset={() => onResetColor(widgetColor.key, tab)}
                 resetDisable={
```

Comment:
https://github.com/rango-exchange/rango-client/assets/123297941/3af7cf0f-2050-4a0f-978b-f8cd95052d5a

When no preset is selected, the reset button doesn't work. I think it worked in the previous version.

Reply:
fixed